keyboards/jones/v.0.3.1/keymaps/default_ansi/keymap.c

```diff
@@ -0,0 +1,372 @@
+        _______,    _______,_______,_______,_______,_______,_______,_______,_______,KC_PGUP,KC_LEFT,KC_RGHT,            KC_VOLD,
+        _______,        _______,_______,_______,_______,_______,_______,_______,_______,KC_PGDN,KC_DOWN,_______,KC_PGUP,_______,
+        _______,_______,_______,_______,        _______,        _______,         _______,_______,_______,KC_HOME,KC_PGDN,KC_END
+    ),
+    [_RAISE] = LAYOUT_ansi(
+        KC_PAUS,KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,  KC_F10, KC_F11, KC_F12, KC_INS, KC_DEL,
+        KC_PSCR,    _______,_______,CT_E,   _______,_______,_______,_______,_______,_______,KC_PGUP,KC_DEL,     _______,_______,
+        _______,    CT_A,   _______,KC_DEL, KC_RGHT,KC_ESC, KC_LEFT,KC_DOWN,KC_UP,  KC_RGHT,KC_MINS,KC_INS,             _______,
+        _______,        _______,_______,_______,_______,KC_LEFT,KC_PGDN,KC_ENT, _______,KC_MRWD,KC_MFFD,_______,KC_PGUP,_______,
+        _______,_______,_______,_______,        _______,        _______,        _______,_______,_______,KC_HOME,KC_PGDN,KC_END
+    ),
+    [_NUM_RAISE] = LAYOUT_ansi(
+        KC_ESC, KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS,KC_EQL, KC_BSLS,KC_GRV,
+        KC_TAB,     KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_BSPC,    KC_LBRC,KC_RBRC,
+        KC_LCTL,    KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN, KC_ENT,            KC_QUOT,
+        S_CAP,          KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT, C_SLSH, KC_RSFT,KC_UP,  _______,
+        _______,_______,_______,_______,        _______,        _______,        _______,_______,_______,_______,_______,_______
+    ),
+    [_ADJUST] = LAYOUT_ansi(
+        _______,RGB_HUI,RGB_SAI,RGB_VAI,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,
+        _______,    _______,WIN,    _______,RESET,  _______,RGB_HUI,RGB_SAI,RGB_VAI,_______,RGB_RMOD,_______,   _______,_______,
+        _______,    AU_TOG, CK_TOGG,MU_TOG, MU_MOD, _______,RGB_HUD,RGB_SAD,RGB_VAD,RGB_TOG,RGB_MOD,_______,            _______,
+        _______,        CK_RST, CK_DOWN,CK_UP  ,_______,_______,NUM,     MAC,   _______,_______,_______,_______,_______,_______,
+        _______,_______,_______,_______,        _______,        _______,        _______,_______,_______,_______,_______,_______
+    )
+};
+
+#if defined(AUDIO_ENABLE) && defined(MUSIC_MAP)
+const uint8_t music_map[MATRIX_ROWS][MATRIX_COLS] = LAYOUT_ansi(
+    51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65,
+    38,   38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48,   49, 50,
+    25,   26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36,       37,
+    11,     12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
+    0,   1,  2,  3,      4,      5,      6,  7,  8,  9, 10, 11
+);
+#endif
+
```

Comment:
Would prefer that this go in the keyboard's c file, so that others can use it without having to explicitly define it.

Reply:
Moved "music_map" to keyboard's c file.
To provide common definition for other keymap creator.